frontend/components/form/FormItem.vue

```diff
@@ -0,0 +1,24 @@
+<!-- SPDX-License-Identifier: AGPL-3.0-or-later -->
+<template>
+  <div class="flex flex-col gap-y-1">
+    <FormLabel :name="name" :label="label" :required="required" :for="id" />
+    <slot v-bind="field" :id="id" />
+    <FormErrorMessage
+      v-if="field.errorMessage"
+      :id="`${id}-error`"
+      :message="field.errorMessage.value"
+    />
+  </div>
+</template>
+<script setup lang="ts">
```

Comment:
Minor formatting request that we do a space between the template and script blocks :) Makes it easier to see that the HTML-ish part has ended :)